app/src/main/java/io/gnosis/safe/di/modules/ApplicationModule.kt

```diff
@@ -272,4 +279,20 @@ class ApplicationModule(private val application: Application) {
         ownerVault: OwnerCredentialsRepository
     ): CredentialsRepository =
         CredentialsRepository(ownerDao, encryptionManager, ownerVault)
+
+    @Provides
+    @Singleton
+    fun providesFirebaseRemoteConfig(): FirebaseRemoteConfig {
+        val remoteConfig = Firebase.remoteConfig
+        val configSettings = remoteConfigSettings {
+            // increase the number of fetches available per hour during development.
+            minimumFetchIntervalInSeconds = if (CLIENT_GATEWAY_URL.contains("staging")) 0 else 3600
```

Comment:
Just to clarify: In a staging environment, every change should be visible on every app start. In production it might take one hour to see changes?

Reply:
yes, that's the idea. Values can be adjusted if needed